crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -161,13 +161,19 @@ impl AuthorityEpochTables {
 impl AuthorityPerEpochStore {
     pub fn new(committee: Committee, parent_path: &Path, db_options: Option<Options>) -> Self {
         let epoch_id = committee.epoch;
-        let tables = AuthorityEpochTables::open(epoch_id, parent_path, db_options);
+
+        let mut epoch_dirname = String::from("epoch_");
+        epoch_dirname.push_str(&epoch_id.to_string());
+        let parent_per_epoch_path = parent_path.join(epoch_dirname);
+
+        let tables =
+            AuthorityEpochTables::open(epoch_id, parent_per_epoch_path.as_path(), db_options);
```

Comment:
Ignore these changes. It's a fix that I put up separately in #6840 that this PR depends on. Will go away once rebased